### DISCUSSION TITLE
Bypass the Quick Setup screen

### DISCUSSION THREAD

**Discussion by ararezaee** [d2phap/ImageGlass#1703] [General] [answered]
Pretty much just the title.

EdgeView installs Edge browser on my PC that's why I don't wanna install it.

**Comment by d2phap** [comment, marked as answer]
You can create `igconfig.default.json` file in the installed folder of ImageGlass with this content:
```json
{
  "_Metadata": {
    "Description": "ImageGlass configuration file",
    "Version": "9.0"
  },
  "QuickSetupVersion": 9
}
```
More details can be found here: https://imageglass.org/news/introducing-the-new-imageglass-version-9-88#breaking-changes
